# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libetpan/0001-fix-searchkey-subject.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From ec8835073ad1da456d007c9a20366968cda32b6b Mon Sep 17 00:00:00 2001
From: Nadeem Shehzad <Nadeem.Shehzad@securepoint.de>
Date: Tue, 25 Jan 2011 12:04:33 +0100
Subject: [PATCH 1/3] fix: searchkey subject corrected by passing correct type "subject" instead of "since".

---
 src/low-level/imap/mailimap_types_helper.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/low-level/imap/mailimap_types_helper.c b/src/low-level/imap/mailimap_types_helper.c
index 7f0934b..b84e542 100644
--- a/src/low-level/imap/mailimap_types_helper.c
+++ b/src/low-level/imap/mailimap_types_helper.c
@@ -800,7 +800,7 @@ mailimap_search_key_new_since(struct mailimap_date * sk_since)
 struct mailimap_search_key *
 mailimap_search_key_new_subject(char * sk_subject)
 {
-  return mailimap_search_key_new(MAILIMAP_SEARCH_KEY_SINCE, NULL, NULL,
+  return mailimap_search_key_new(MAILIMAP_SEARCH_KEY_SUBJECT, NULL, NULL,
 				 NULL, NULL, NULL, NULL, NULL, NULL,
 				 sk_subject, NULL, NULL, NULL, NULL,
 				 NULL, 0, NULL,
-- 
1.6.3.3

